en/docs/install-and-setup/setup/distributed-deployment/product-profiles.md

```diff
@@ -158,6 +158,9 @@ Create an optimized distribution for a particular API-M profile.
 
 #### Method 2 - Optimizing while starting the server
 
+!!! note
+    Before running these profiles, take the backup of `<PRODUCT-HOME>/repository/resources/conf/templates/repository/conf/axis2` and `<PRODUCT-HOME>/ repository/resources/conf/templates/repository/conf/registry_TM.xml.j2` directories.
```

Comment:
Shall we change this as

Before running these profiles, take the backup of `<PRODUCT-HOME>/repository/resources/conf/templates/repository/conf/axis2` directory and `<PRODUCT-HOME>/ repository/resources/conf/templates/repository/conf/registry_TM.xml.j2` file

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/6215/commits/0c0f0becf96542fbc506712404aef53efa128892